CLI/src/main/java/main/Drifty_CLI.java

```diff
@@ -189,6 +219,172 @@ public static void main(String[] args) {
         }
     }
 
+    private static void ensureYamlFileExists() {
+        // Check if the YAML file exists, create it if it does not
+        File yamlFile = new File(yamlFilePath);
+        if (!yamlFile.exists()) {
+            try {
+                boolean isNewFileCreated = yamlFile.createNewFile();
+                if (isNewFileCreated) {
+                    messageBroker.msgFilenameInfo("New YAML file created: " + yamlFilePath);
```

Comment:
@Femi-lawal We should rather keep this statement in the logs because the user has nothing to do with this :smile:.
```suggestion
                    messageBroker.msgLogInfo("New YAML file created: " + yamlFilePath);
```